pkg/controller/ingress/concurrency_watchdog.go

```diff
@@ -135,11 +141,17 @@ func (c *ConcurrencyWatchdog) Start(ctx context.Context) error {
 
 func (c *ConcurrencyWatchdog) tick(ctx context.Context) error {
 	start := time.Now()
+	var retErr *multierror.Error
 	defer func() {
 		c.logger.Info("finished checking on ingress controller pods", "latencySec", time.Since(start).Seconds())
+
+		//placing this call inside a closure allows for result and err to be bound after tick executes
```

Comment:
I wonder if we should maybe just include this comment once in a README or something similar in the controller dir - that way we don't have to repeat it so frequently. 

open to leaving it though if we think that's the cleanest solution, just not a fan of how many times we repeat it